From dcfea54787a3efe1f1ab44eef998c2a7a723d80a Mon Sep 17 00:00:00 2001
From: Peter Hutterer <peter.hutterer@who-t.net>
Date: Tue, 18 Sep 2018 07:02:27 +1000
Subject: [PATCH] quirks: extend/restore the Elan Touchpads quirks

The hwdb match entry used to be this one:
 libinput:name:*Elan Touchpad*:dt:*
  LIBINPUT_ATTR_PRESSURE_RANGE=10:8
from commit 596777a3147d4070f9598f051f6d7ae96fa2f49a. It was intended to match
for devicetree only but the way the udev rules were composed, it ended up
matching on any system.

Restore that for all systems to have compatibility with 1.11. For this one,
let's also add the resolution hint and hope that that works too.

Fixes #140

Signed-off-by: Peter Hutterer <peter.hutterer@who-t.net>
---
 quirks/30-vendor-elantech.quirks | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/quirks/30-vendor-elantech.quirks b/quirks/30-vendor-elantech.quirks
index e49c3415..9b3a1ba7 100644
--- a/quirks/30-vendor-elantech.quirks
+++ b/quirks/30-vendor-elantech.quirks
@@ -4,3 +4,8 @@
 MatchName=*Elantech Touchpad*
 AttrResolutionHint=31x31
 AttrPressureRange=10:8
+
+[Elan Touchpads]
+MatchName=*Elan Touchpad*
+AttrResolutionHint=31x31
+AttrPressureRange=10:8
-- 
2.18.0


